clang-tools-extra/clangd/ConfigCompile.cpp

```diff
@@ -504,10 +504,10 @@ struct FragmentCompiler {
       auto Fast = isFastTidyCheck(Str);
       if (!Fast.has_value()) {
         diag(Warning,
-             llvm::formatv(
-                 "Latency of clang-tidy check '{0}' is not known. "
-                 "It will only run if ClangTidy.FastCheckFilter is Loose or None",
-                 Str)
+             llvm::formatv("Latency of clang-tidy check '{0}' is not known. "
```

Comment:
nit: I'm seeing some diff hunks containing unrelated formatting changes. If would be great to remove this as they pollute the blame

Reply:
I use `clang-format` with the `.clang-format` of the code base. Even if I don't change anything and just do a `clang-format` this change will happen. It seems that it's not correctly formatted. I will fix this and my recommendation would be to use `clang-format` on this file - or perhaps the whole code base - post patching in its own commit. this will fix this issue(s) in the future.